api/src/main/java/org/openmrs/module/poc/api/common/util/OPENMRSUUIDs.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.poc.api.common.util;
+
+public interface OPENMRSUUIDs {
```

Comment:
change this from interface to class and all constants should be public static final. This an example of how openmrs does and we would like to maintain those 
**public static final String APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY = "application_data_directory";**
When calling them you just use class and the property

Reply:
class name could be
public class PocConstants{
public static final String DEFAULT_ENCOUNTER_ROLE_UUID = "a0b03050-c99b-11e0-9572-0800200c9a66";
.
.
.
}

Reply:
in openmrs that is refereed to as shown below
https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/EncounterRole.java#L20